demo/web/src/main/notebooks/C. Python/C1 Python lists.md

```diff
@@ -0,0 +1,34 @@
+# Python lists
+
+This notebook will demonstrate how to use Python lists within Deephaven.
+
+Lists are native Python data structures that can be declared using the bracket `[ ]` syntax.
+
+```python
+my_list = [1,2,3]
+```
+
+Lists can be used just like other objects within a query. When using a list within a query, Deephaven will automatically assign its column to the `org.jpy.PyListWrapper` type.
+
+```python
+from deephaven.TableTools import emptyTable
+
+list_table = emptyTable(1).update("List = my_list")
+```
+
+When using a custom function to generate a list, you need to cast the function call to `org.jpy.PyListWrapper` in order for Deephaven to recognize the return value as a list and assign the column type properly. Otherwise it will be defined as a generic object and won't be useable as expected.
```

Comment:
```suggestion
When using a custom function to generate a list, you need to cast the function call to `org.jpy.PyListWrapper` in order for Deephaven to recognize the return value as a list and assign the column type properly. Otherwise, it will be defined as a generic object and won't be useable as expected.
```